hw4.Rmd

```diff
@@ -30,43 +37,103 @@ Before doing anything else, just play with the data to get a sense of it.
 (@) The main outcome of interest is *decision* which records whether the judge granted an appeal or not.  In what proprotion of cases did the judge grant an appeal.
 
 ```{r}
-# [your code here]
+# languages less likely to have an appeal granted?
```

Comment:
Nice visualization, but did you answer the question about the proportion of cases that the judges granted appeal.